plugins/telemetry-plugin/package.json

```diff
@@ -0,0 +1,45 @@
+{
+      "name": "@eclipse-che/telemetry-plugin",
+      "version": "0.0.1",
+      "publisher": "Eclipse Che",
+      "keywords": [
+        "theia-plugin"
+      ],
+      "description": "Manage Eclipse Che Telemetry Plugin",
+      "license": "EPL-2.0",
+
+      "files": [
+        "src"
+      ],
+      "activationEvents": [
+        "*"
+      ],
+      "devDependencies": {
+        "@theia/plugin": "next",
+        "@theia/plugin-packager": "latest",
+        "@eclipse-che/plugin": "0.0.1",
+        "rimraf": "2.6.2",
+        "typescript-formatter": "7.2.2",
+        "typescript": "2.9.2",
+        "ts-loader": "^4.1.0",
+        "clean-webpack-plugin": "^0.1.19",
```

Comment:
could you please remove unneeded dependencies: about webpack*

Reply:
removed